### PR TITLE
Bugfix/scrollbar

### PR DESCRIPTION
**Pull request by PradeepRajput07**
Hero Banner: Made the DesktopAsset field as required/mandatory to be entered

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/herobanner/_cq_dialog/.content.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
                                 <items jcr:primaryType="nt:unstructured">
                                     <desktopasset
                                         jcr:primaryType="nt:unstructured"
-                                        sling:resourceType="cq/gui/components/authoring/dialog/fileupload"
+                                        sling:resourceType="granite/ui/components/foundation/form/fileupload"
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/herobanner/_cq_dialog/.content.xml]: Change it to the "granite/ui/components/coral/foundation/form/fileupload"

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/herobanner/_cq_dialog/.content.xml [hunk 2]
```diff
@@ -34,13 +34,14 @@
...
                                         name="./desktopAsset"
+                                        required="{Boolean}true"
                                         rootPath="/content/dam"
                                         title="Upload Image or Video"
                                         uploadUrl="${suffix.path}"
                                         useHTML5="{Boolean}true"/>
                                     <mobileimage
                                         jcr:primaryType="nt:unstructured"
-                                        sling:resourceType="cq/gui/components/authoring/dialog/fileupload"
+                                        sling:resourceType="granite/ui/components/foundation/form/fileupload"
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/herobanner/_cq_dialog/.content.xml]: Change it to the "granite/ui/components/coral/foundation/form/fileupload"

Comment (PradeepRajput07) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/herobanner/_cq_dialog/.content.xml]: That coral one doesn't work and is not rendered properly. So only using this one

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/herobanner/_cq_dialog/.content.xml (hunk 1 above)
Comment (PradeepRajput07) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/content/herobanner/_cq_dialog/.content.xml]: That coral one doesn't work and is not rendered properly. So only using this one

**Pull request merged**